using_images/xpaas_images/a_mq.adoc

```diff
@@ -5,4 +5,92 @@
 :icons:
 :experimental:
 
-Information on the Red Hat JBoss A-MQ xPaaS image will be added soon.
+Red Hat JBoss A-MQ (JBoss A-MQ) is available as a containerized xPaaS image that is designed for use with OpenShift v3. It allows developers to quickly deploy an A-MQ message broker in a hybrid cloud environment.
+
+[IMPORTANT]
+There are significant differences in supported configurations and functionality in the JBoss A-MQ image compared to the regular release of JBoss A-MQ.
+
+This topic details the differences between the JBoss A-MQ xPaaS image and the regular release of JBoss A-MQ, and provides instructions specific to running and configuring the JBoss A-MQ xPaaS image. Documentation for other JBoss A-MQ functionality not specific to the JBoss A-MQ xPaaS image can be found in the https://access.redhat.com/documentation/en-US/Red_Hat_JBoss_A-MQ/[JBoss A-MQ documentation on the Red Hat Customer Portal].
+
+== Differences Between the JBoss A-MQ xPaaS Image and the Regular Release of JBoss A-MQ
+
+There are several major functionality differences in the OpenShift JBoss A-MQ xPaaS image:
+
+* The Karaf shell is not available.
+* The Fuse Management Console (Hawtio) is not available.
+* Configuration of the broker can be performed:
+** using parameters specified in the A-MQ application template, as described in xref:configuring-params[Application Template Parameters].
+** using the STI (Source-to-image) tool, as described in xref:configuring-sti[Configuration Using STI].
+
+
+== Installing the JBoss A-MQ xPaaS Images
+
+To use the Red Hat xPaaS middleware images in your OpenShift project, you must first install the image streams and source-to-image (S2I) application templates.
+
+. In a terminal, change into the root directory of your OpenShift master. It should contain the 'examples' subdirectory.
+. Create the image stream from which the A-MQ image will be obtained:
++
+----
+# oc create -f examples/application-templates/jboss-image-streams.json -n <project>
+----
+. Create the S2I templates for your A-MQ application:
++
+----
+# oc create -f examples/application-templates/amq/amq6.json -n <project>
+# oc create -f examples/application-templates/amq/amq6-persistence.json -n <project>
+----
+
+You will now have the JBoss A-MQ xPaaS images stream and S2I application templates available for use in your OpenShift project. 
+
+To create an application based on the template, follow instructions in the *Creating a Configuration From a Template* section of the link:../dev_guide/templates.html[Creating New Applications] topic in the OpenShift documentation. For a reference on template parameters specified when creating the application, see the xref:configuring-params[Application Template Parameters] section below.
```

Comment:
Bad link:

```
link:../dev_guide/templates.html[Creating New Applications]
```

Should go up one more level.

Also, the topic title and section seem to be off or out of date. Were you meaning to link to `new_app.html#specifying-a-template`? I.e, the "Specifying a Template" section of the "Creating New Applications" topic. Or was it meant to be `templates.html#creating-resources-from-a-template` for the "Creating Resources from a Template" section of the "Templates" topic?